tests/capture/test_capture_cond.py

```diff
@@ -849,21 +849,16 @@ def f(x):
         qml.assert_equal(q.queue[1].base, qml.RX(0.5, 0))
 
 
+@pytest.mark.usefixtures("enable_disable_dynamic_shapes")
 def test_cond_abstracted_axes():
```

Comment:
I think you can use the fixture as a test function argument. Same comment for other files where only 1-2 tests are added:
```suggestion
def test_cond_abstracted_axes(enable_disable_dynamic_shapes):
```